Source/Core/DolphinQt2/Settings/PathPane.cpp

```diff
@@ -39,7 +40,7 @@ void PathPane::Browse()
 void PathPane::BrowseDefaultGame()
 {
   QString file = QFileDialog::getOpenFileName(
-      this, tr("Select a Game"), QDir::currentPath(),
+      this, tr("Select a Game"), QString::fromStdString(SConfig::GetInstance().m_strDefaultISO),
```

Comment:
Given `SConfig::GetInstance().m_strDefaultISO` is used twice, perhaps make a reference to it like:

```cpp
std::string& default_iso = SConfig::GetInstance().m_strDefaultISO;
```

to avoid duplicating the slightly long call through SConfig.